cmd/livepeer/livepeer.go

```diff
@@ -379,6 +380,19 @@ func main() {
 		// TODO provide an option to disable this?
 		*rtmpAddr = defaultAddr(*rtmpAddr, "127.0.0.1", RtmpPort)
 		*httpAddr = defaultAddr(*httpAddr, "127.0.0.1", RpcPort)
+		if *webhookURL != "" {
```

Comment:
Perhaps factor this out into a `getWebhookURL` helper? Eg, `server.WebhookURL = getWebhookURL(*webhookURL)`

Reply:
done